pkgs/tools/security/echidna/default.nix

```diff
@@ -1,5 +1,4 @@
 { lib
```

Comment:
5. Is there some reason echidna is not on Hackage?  Could you upload it to hackage?  That way, you wouldn't have to maintain this file here, but it would get updated automatically with the rest of the Haskell packages in Nixpkgs.

Reply:
I have to speak with the rest of the echidna team, but I think we can do this.

Reply:
Is it okay if we merge this and I'll work on hackage for the next update?

Reply:
Yeah, that sounds good, thanks!